compiler_opt/es/blackbox_optimizers.py

```diff
@@ -384,8 +406,13 @@ def mc_gradient(precision_parameter,
   return gradient
 
 
-def sklearn_regression_gradient(clf, est_type, perturbations, function_values,
-                                current_value):
+def sklearn_regression_gradient(
+    clf: Union[linear_model.Ridge,
```

Comment:
same thing with est_type (maybe add the todo here so it's easy to find when we refactor)

Reply:
or maybe the issue can say there are many instances of this (noticed more down below)

Reply:
The types for clf, est_type, and hyperparametes_update_method have been converted to enums